File: code/c/control-flow/test-compound.c
/* Program: test-compound.c */

int main()
{
    char ch;
    
    printf("Enter the forest? [y/N]: ");
    scanf("%c", &ch);
    
    if (ch == 'y' || ch == 'Y')
    {
        printf("We are the Knights who say 'Ni'!\n"); 
        printf("We are the keepers of the sacred words: ");
        printf("'Ni', 'Peng', and 'Neee-wom'!\n"); 
        printf("The Knights Who Say 'Ni' demand a sacrifice.\n"); 
        printf("We want... a shrubbery!\n");
    }
    else
    {
        printf("Greetings Sir Robin.\n");
    }
}

File: code/pascal/control-flow/TestCompound.pas
program TestCompound;

procedure Main();
var
  ch: Char;
begin
    Write('Enter the forest? [y/N]: ');
    ReadLn(ch);
    
    if (ch = 'y') or (ch = 'Y') then
    begin
        WriteLn('We are the Knights who say "Ni"!'); 
        WriteLn('We are the keepers of the sacred words: ');
        WriteLn('"Ni", "Peng", and "Neee-wom"!'); 
        WriteLn('The Knights Who Say "Ni" demand a sacrifice.'); 
        WriteLn('We want... a shrubbery!');
    end
    else
    begin
        WriteLn('Greetings Sir Robin.');
    end;
end;

